dotnet/src/Extensions/Reliability.Polly/HttpRetryConfig.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Reliability.Polly;
+
+/// <summary>
+/// Retry configuration for DefaultKernelRetryHandler that uses RetryAfter header when present.
+/// </summary>
+public sealed record HttpRetryConfig
```

Comment:
It feels very odd to use this in conjunction with Polly when it already has its own configuration mechanisms. What is the goal here with retry support?

Reply:
To not break current usage of Kernel Configuration file, (intermediate translation to Polly policies).

We might Obsolete this in the long run.

Reply:
But today kernel configuration is used only for the DefaultHttpRetryPolicy -- never related to Polly implementations of things. I really was hoping to keep the implementation, just in its own package, and not be related to Polly.